axolotl-web/src/pages/MessageList.vue

```diff
@@ -135,6 +170,13 @@ export default {
       lastCHeight: 0,
       lastMHeight: 0,
       scrollLocked: false,
+      recorder: null,
+      recording: false,
+      recorded: false,
+      playing: false,
+      blobUrl: "",
+      voiceNoteElem: null,
+      duration:0,
```

Comment:
Does it work to group properties in `data`? Like:
```js
data() {
  return {
    ...,
    voiceNote: {
      recorder,
      ...
    }
}
```

Reply:
good idea!